src/reconcile/rec_visibility.c

```diff
@@ -981,12 +981,9 @@ __wti_rec_upd_select(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_INSERT *ins,
     if (upd_select->tombstone != NULL)
         F_SET(upd_select->tombstone, WT_UPDATE_DS);
 
-    /*
-     * Set statistics for update restore evictions. Update restore eviction debug mode forces update
-     * restores to both committed or uncommitted changes.
-     */
-    if (supd_restore || F_ISSET(r, WT_REC_SCRUB))
-        r->cache_write_restore = true;
+    /* Track whether we need to do update restore eviction. */
+    if (supd_restore)
+        r->cache_write_restore_uncommitted = true;
```

Comment:
wouldn't `supd_restore` be true even for committed updates that are not visible to this txnid?

Reply:
yes, uncommitted is relative to the current transaction's snapshot. If you don't like the name, we can change it to something else.

Reply:
May be cache_write_restore_invisible

Reply:
Yes, r->cache_write_restore_invisible or something like that. Also, I am curious to know the motivation to make it true when `WT_REC_SCRUB` is set?